internal/tsc_wrapped/compiler_host.ts

```diff
@@ -332,6 +333,28 @@ export class CompilerHost implements ts.CompilerHost, tsickle.TsickleHost {
       }
     }
 
+    // TODO(gmagolan): clean this up
```

Comment:
This is basically doing a reverse lookup from the tsconfig paths to figure out what the module name should be. When building angular directly from source not all imports are scoped to the module name. The .ngsummary and .ngfactory files from the angular packages are not.